spec/services/reject_application_by_default_spec.rb

```diff
@@ -29,6 +29,22 @@ def create_application
     expect(application_choice.reload.rejected_by_default).to be true
   end
 
+  it 'sets `rejected_at`' do
```

Comment:
Same as above.

Reply:
I think this belongs here, it is RejectApplicationByDefault's responsibility to set ```rejected_at```, which SetDeclineByDefault then uses to determine whether decisions are pending or what the most recent decision is. SetDeclineByDefault only reads these attributes, doesn't write them. The are written by:

- MakeAnOffer sets ```offerred_at```
- RejectApplication sets ```rejected_at```
- RejectApplicationByDefault sets ```rejected_at``` (as well as ```rejected_by_default```)